scripts/promote_staging_to_prod.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+checks=$(curl -s  "https://api.github.com/repos/chanzuckerberg/single-cell-data-portal/actions/runs?actor=czibuildbot" | jq -r '.workflow_runs[] | select(.display_title == "stage") | .conclusion' | head -n 1)
+if [ "$checks" == "success" ]; then
+  echo "All checks passed in most recent staging deployment!"
+else
+  echo "Some checks failed in most recent staging deployment. Please check GHA runs."
+  exit 1
+fi
+
+echo "fetch origin branch history"
+git fetch
+echo "Stash any local changes"
+git stash
```

Comment:
I don't totally like having this script mutate the local working tree. I think, in order of preference:
1) fail with message if the working tree is not clean
or
2) pop the stash at the end after stashing